examples/InputRotary/InputRotary.ino

```diff
@@ -17,19 +18,19 @@ const char* charset = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
 void inputCallback(char* value);
 void clearInput();
 
-extern MenuItem* usernameMenu[];
+// clang-format off
+MENU_SCREEN(userScreen, userItems,
+    ITEM_INPUT_CHARSET("User", charset, inputCallback),
+    ITEM_COMMAND("Clear", clearInput));
+// clang-format on
 
-MAIN_MENU(
-    ITEM_SUBMENU("Set user", usernameMenu),
+// clang-format off
+MENU_SCREEN(mainScreen, mainItems,
+    ITEM_SUBMENU("Set user", userScreen),
     ITEM_BASIC("Settings"),
     ITEM_BASIC("More Settings"),
     ITEM_BASIC("And more Settings"));
-
```

Comment:
Can the screens not be ordered as they appear, I imagine this would confuse anyone reading the code.
I would expect the main menu to be at the top, and submenus follow.

Reply:
It's possible if declare screens first